pcsx2/gui/FrameForGS.cpp

```diff
@@ -155,11 +155,22 @@ void GSPanel::DoResize()
 
 	double clientAr = (double)client.GetWidth()/(double)client.GetHeight();
 
+	extern AspectRatioType iniAR;
+	extern bool switchAR;
 	double targetAr = clientAr;
-	if( g_Conf->GSWindow.AspectRatio == AspectRatio_4_3 )
```

Comment:
FMVstarted always runs for as long as you run the FMV and affects both the FMV 43 switch and FMV software switch, switchAR is just for the FMV 43 switch, and can be turned off if you want to manually change the aspect ratio during a FMV.